app/views/admin/instances/show.html.haml

```diff
@@ -114,6 +114,16 @@
 - if @instance.persisted?
   %hr.spacer/
 
+  %h3= t('admin.instances.audit_log.title')
+  - if @action_logs.empty?
+    %p= t('accounts.nothing_here')
+  - else
+    .report-notes
+      = render partial: 'admin/action_logs/action_log', collection: @action_logs
+    = link_to t('admin.instances.audit_log.view_all'), admin_action_logs_path(target_domain: @instance.domain), class: 'button'
```

Comment:
Only persisted instances can have action logs, I believe, since there needs to be something present for it to show, I think?

I'm not super keen on reusing `accounts.nothing_here` in this view, but it's the string that's used everywhere else when we've no content for something.

I'm not using the `nothing_here` helper here because of the styling of that being designed more for tables than just a paragraph of text.